sklearn/multioutput.py

```diff
@@ -0,0 +1,151 @@
+"""
+This module implements multioutput regression.
```

Comment:
I don't really think this needs a separate module, but I do get there's an issue of increasingly inaccurate naming of `multiclass.py` if included there.